components/AudioSettings/PlaySound.js

```diff
@@ -0,0 +1,23 @@
+import { useRef, useEffect } from "react";
+import memory from "@/public/sounds/memory.mp3";
+
+export default function PlaySound({
+  audioTrigger,
+  setCurrentVolume,
+  currentVolume,
+  page,
+  isMuted,
+}) {
+  const audioReference = useRef(null);
+
+  useEffect(() => {
+    const audio = audioReference.current;
+    if (audioTrigger === true && audio) {
```

Comment:
this should also work like this:

```suggestion
    if (audioTrigger && audio) {
```